### ISSUE TITLE
tabulate: --wrap-column fails when --split-comma record exists

### ISSUE THREAD

**Issue by lmorg** [lmorg/murex#259]
```
 -t, --timeout TIMEOUT      Use this timeout in seconds for container shutdown
                               when attached or when containers are already
                               running. (default: 10
```
 produces:

```
» docker-compose: help up -> tabulate: --split-comma --map --column-wraps
{
    "--abort-on-container-exit": "Stops all containers if any container was stopped. Incompatible with -d. when attached or when containers are already running. (default: 10)",
    "--build": "Build images before starting containers.",
    "--exit-code-from SERVICE": "Return the exit code of the selected service container. Implies --abort-on-container-exit.",
    "--force-recreate": "Recreate containers even if their configuration and image haven't changed. Incompatible with --no-recreate.",
    "--no-build": "Don't build an image, even if it's missing.",
    "--no-color": "Produce monochrome output.",
    "--no-deps": "Don't start linked services.",
    "--no-recreate": "If containers already exist, don't recreate them. Incompatible with --force-recreate.",
    "--no-start": "Don't start the services after creating them.",
    "--remove-orphans": "Remove containers for services not defined in the Compose file",
    "--scale SERVICE=NUM": "Scale SERVICE to NUM instances. Overrides the `scale` setting in the Compose file if present.",
    "--timeout TIMEOUT": "Use this timeout in seconds for container shutdown",
    "-d": "Detached mode: Run containers in the background, print new container names. Incompatible with --abort-on-container-exit.",
    "-t": "Use this timeout in seconds for container shutdown"
}
```
(please not the `-t` and `--timeout` values)

**Comment by lmorg** [comment]
issue lies here:
```
		if splitComma && len(split) > 1 {
			comma := strings.Split(split[0], ",")
			if len(comma) == 1 {
				goto noSplit
			}

			for i := range comma {
				flag := strings.TrimSpace(comma[i])
				new := append([]string{flag}, split[1:]...)
				err := w.Write(new)
				if err != nil {
					return err
				}
			}
			continue
		}
```
I shouldn't be `w.Write`ing here...

**Comment by lmorg** [comment]
Fixed in 78afc350518dee9370547f9385996a31290954e0
